Dockerfile

```diff
@@ -1,8 +1,20 @@
-FROM python:3-alpine@sha256:3998e97aeeabfdeea57c6e05c7544c460ae158f24cc74371b7d4eca18fbc3171
+FROM ghcr.io/radiorabe/s2i-python:0.4.4 AS build
```

Comment:
Will we have to update this reference by hand or does dependabot that for us?

Reply:
dependabot will update it once daily

with most other containers i've found that i usually trigger dependabot manually just because i don't want to wait until it decides to run on it's own

i'll wait with merging this until next month, there are a lot of thing already in the pipeline due to me forgetting to run the image from ghcr.io instead of docker hub and this change is huge in itself